lib/manager/components/ProjectFeedListToolbar.js

```diff
@@ -231,6 +236,20 @@ export default class ProjectFeedListToolbar extends PureComponent<Props> {
     this.props.setFeedSort(type)
   })
 
+  _onLabelClick = (labelId: string) => {
+    // This variable must be set directly from this.props both
+    // to allow the override and for flow
+    const labels = this.props.filter.labels || []
```

Comment:
If we just set the default reducer state for filter.labels to `[]`, we shouldn't have to introduce this conditional here.

Reply:
Much cleaner thank you!